plugin.yml

```diff
@@ -1,7 +1,7 @@
 name: DevTools
 main: DevTools\DevTools
 version: 1.12.4
-api: [3.0.0-ALPHA7, 3.0.0-ALPHA8]
+api: [3.0.0-ALPHA7, 3.0.0-ALPHA8, 3.0.0-ALPHA9]
```

Comment:
I don't think this has an ALPHA9 requirement? It can be pushed to ALPHA8 without any problems

Reply:
Not sure how your bump commit snuck in there...